proparse/src/main/java/org/prorefactor/treeparser/SymbolScope.java

```diff
@@ -34,7 +31,7 @@
  * record or frame scoping!
  */
 
-public class SymbolScope implements Xferable {
+public class SymbolScope {
   private static final Integer DATASET = new Integer(NodeTypes.DATASET);
   private static final Integer DATASOURCE = new Integer(NodeTypes.DATASOURCE);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this "Integer" constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2129)